src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -357,19 +359,36 @@ func (cfg Config) doFetchCloudConfig(proxyAddr string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Unable to initialize HTTP client: %s", err)
 	}
+	country := strings.ToLower(globals.GetCountry())
```

Comment:
This relies on globals.go SetLocation having already been called at this point right? Otherwise GetCountry will return nil.